### PR TITLE
PR for journal-week-6.md

### PR DESCRIPTION
**Pull request by jeremyrist**

### CODE REVIEW

File: tutorial-week-6.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+_How to do it:_
+
+1. Place your image in your `html`
+
+2. Give your image a height and width in `scss`
+
+3. Make sure that the height and width are the same!
+
+4. Change the border-raidus to 50%
```
Comment (s4dmw) [inline tutorial-week-6.md]: check that spelling on "radius", yo!

**Comment by s4dmw** [comment]
useful info and concise  :+1: 

see my line note for a minor spelling error...

**Comment by rosalsm** [comment]
I like it, simple, easy and useful :+1:

**Comment by boshnivolo** [comment]
Super useful! I need this for my blog! :+1:

**Pull request merged**